### DISCUSSION TITLE
Reference field is disabled initially

### DISCUSSION THREAD

**Discussion by fabien** [sanity-io/sanity#3391] [Sanity Studio Support] [answered]
When no reference has been assigned yet, a field of type `reference` is disabled initially:

<img width="606" alt="image" src="https://user-images.githubusercontent.com/2749/177103335-c565f98f-41e5-497c-b4cd-393bad6fbb4f.png">

It's only after clicking 'Create new' and removing that draft again, that the search/autocomplete field will become enabled, in order to select a pre-existing item from the list.

**Comment by kmelve** [comment, marked as answer]
This seems like a bug! Thanks for reporting 🙇

**Reply by kmelve** [reply]
Issue tracked here https://github.com/sanity-io/sanity/issues/3495
